release-notes/8.0/preview/8.0.0-preview.3.md

```diff
@@ -6,7 +6,7 @@ The .NET 8.0.0 Preview 3 and .NET SDK 8.0.100-preview.3.23178.7 releases are ava
 
 .NET 8 is the next major release of .NET following .NET 8.0. You can see some of the new features available with .NET 8 Preview 3 at [dotnet/core #8135](https://github.com/dotnet/core/issues/8135).
 
-See the [.NET][dotnet-blog], [EF Core][ef-blog] and [ASP.NET Core][aspnet-blog] blogs for additional details.
```

Comment:
(small thing) if the "variable link" is no longer used, it _might_ be removed from the list at the bottom (line 93) too :)